src/groupeddataframe/groupeddataframe.jl

```diff
@@ -517,7 +539,8 @@ Base.haskey(gd::GroupedDataFrame, key::Union{Signed,Unsigned}) =
 Get a group based on the values of the grouping columns.
 
 `key` may be a `NamedTuple` or `Tuple` of grouping column values (in the same
-order as the `cols` argument to `groupby`).
+order as the `cols` argument to `groupby`). It may also be a `Dict`, with keys that are 
+either `Symbol`s or strings, in which case the order of the arguments does not matter. 
```

Comment:
```suggestion
order as the `cols` argument to `groupby`). It may also be a `Dict` with
`Symbol` or string keys, in which case the order of the arguments does not matter. 
```